plugin/providers/phpipam/resource_phpipam_first_free_address.go

```diff
@@ -138,8 +138,8 @@ func resourcePHPIPAMFirstFreeAddressCreate(d *schema.ResourceData, meta interfac
 		}
 		//addrs := d.Get("ip_address")
 
-		if len(addrs) != 1 {
-			return errors.New("IP address either missing or multiple results returned by reading IP after creation")
+		if len(addrs) == 0 {
```

Comment:
Here we can still get multiple addresses
I think it's better to leave the previous condition

Reply:
> Here we can still get multiple addresses I think it's better to leave the previous condition or extent condition with the new condition that check for `0`

Actually, the all idea is that is transparent to the end user, it just makes the use of the subnet explicit and in case IP in two subnets it will use the correct one.  So in fact it does not change the use case at all , just make it more specific